Spanner/src/Backup.php

```diff
@@ -212,7 +215,7 @@ public function create($database, \DateTimeInterface $expireTime, array $options
      * @return LongRunningOperation<Backup>
      * @throws \InvalidArgumentException
      */
-    public function createCopy($newBackup, $expireTime, array $options = [])
+    public function createCopy(Backup $newBackup, \DateTimeInterface $expireTime, array $options = [])
```

Comment:
I think we should probably add `use DateTimeInterface` at the top of the file to make this and the docblocks cleaner